CHANGELOG.md

```diff
@@ -9,7 +9,7 @@ This file is used to list changes made in each version of the AWS ParallelCluste
 **ENHANCEMENTS**
 - Add support for Ubuntu 20.04.
 - Add support for using FSx Lustre in subnet with no internet access.
-- Add support for Centos 7 ARM.
+- Add support for Centos 7 ARM, add FSx support.
```

Comment:
Minor: we don't need to update the changelog with every commit if it's already covered by the main enhancement "Add support for Centos 7 ARM"

We can add the `skip-changelog` tag to PR to skip the changelog CI

Reply:
Thanks!